### PR TITLE
[WIP] Set up Heroku CI for integration tests

### PR DESCRIPTION
**Pull request by anthotsang**
https://app.asana.com/0/683768070860536/817501498200998

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "dblink"
+  enable_extension "pg_stat_statements" unless Rails.env.test?
```
Comment (hanchang) [inline db/schema.rb]: Er should this be `enable_extension "dblink" unless Rails.env.test?`

File: app/models/news_item.rb [hunk 2]
```diff
@@ -135,6 +135,11 @@ def link_coin_from_feedsource
   end
 
   def notify_news_tagger
+    # Don't want to notify news tagger when running tests
```
Comment (hanchang) [inline app/models/news_item.rb]: Great catch!

File: app.json [hunk 3]
```diff
@@ -68,5 +68,16 @@
...
     "scheduler:standard"
   ],
-  "buildpacks": [{ "url": "heroku/nodejs" }, { "url": "heroku/ruby" }]
+  "buildpacks": [{ "url": "heroku/nodejs" }, { "url": "heroku/ruby" }],
+  "environments": {
+    "test": {
+      "formation": {
+        "test": {
+          "quantity": 1,
+          "size": "performance-l"
```
Comment (hanchang) [inline app.json]: Have we tried using a Performance M dyno? What were results like?

File: db/schema.rb (hunk 1 above)
Comment (anthotsang) [reply db/schema.rb]: Yeah, it definitely should be, thanks!

**Comment by hanchang** [comment]
Gonna close this; we already extracted out non-Heroku associated test changes here:
https://github.com/coinfi/coinfi/pull/487

We'll be evaluating other CI vendors shortly.

**Pull request closed without merging**